lib/Quantization/Base/Base.cpp

```diff
@@ -389,6 +391,31 @@ chooseQuantizationParams(TensorProfilingParams profParams, Schema schema,
   min = std::max(min, std::numeric_limits<float>::lowest());
   max = std::min(max, std::numeric_limits<float>::max());
 
+  // Calibrate the min/max range (for non-zero ranges only).
+  if ((profParams.min != profParams.max) && (min != max)) {
+    if (calibration == Calibration::KLMinimization) {
```

Comment:
nit: combine into a single if so there's one less level of nesting?

Reply:
Done.